src/shortcuts/__tests__/undo.ts

```diff
@@ -0,0 +1,41 @@
+import { importText } from '../../action-creators'
+import { NOOP, RANKED_ROOT, ROOT_TOKEN } from '../../constants'
+import { exportContext } from '../../selectors'
+import { createTestStore } from '../../test-helpers/createTestStore'
+import executeShortcut from '../../test-helpers/executeShortcut'
+import undoShortcut from '../undo'
+
+const event = { preventDefault: NOOP } as Event
+
+it('Undo thought change', async () => {
+
+  const store = createTestStore()
+
+  // import thoughts
+  store.dispatch(importText(RANKED_ROOT, `
+  - a
+  - b`))
+
+  store.dispatch({ type: 'setCursor', thoughtsRanked: [
+    { value: 'a', rank: '0' },
+    { value: 'b', rank: '1' }
+  ] })

```

Comment:
You are setting the cursor to `a/b`, but `b` is a child of the root, not `a`. Do you mean `[{ value: 'b', rank: 1 }]`?